cf7-to-google-sheet-extension.php

```diff
@@ -129,41 +148,47 @@ public function editor_panel( $panels ) {
 		}
 
 		/**
-		* Function Name: spreadsheet_editor_panel
-		* Function Description: Saving google spreadsheet data
-		* @param object $post
-		*/
-
-		public function spreadsheet_editor_panel( $post ) { 
-			$form_id   = sanitize_text_field( $_GET['post'] );
-			$form_data = get_post_meta( $form_id, 'cf7_to_spreadsheet_data' );
-			ob_start();
-		    include_once ( plugin_dir_path( __FILE__ ).'include/cf7-editor-panel-html.php' );
-		    $editor_panel_html = ob_get_contents();
-			ob_end_clean();
-		    echo $editor_panel_html;
-		}
-
-		/**
 		* Function Name: setting_page
 		* Function Description: Register google access code
 		*/
 
-		public function setting_page() {
-			include_once ( plugin_dir_path( __FILE__ ).'include/cf7-setting-page-html.php' );
+		public function setting_page( $post ) {
 			ob_start();
+			require ( plugin_dir_path( __FILE__ ).'include/cf7-setting-page-html.php' );
 			$setting_page_html = ob_get_contents();
 			ob_end_clean();
 		    echo $setting_page_html;
+		}
+
+		/**
+		* Function Name: pre_authentication
+		* Function Description: Pre-authentication for generating access code
+		*/
+		public function pre_authentication() {
 			if ( isset ( $_GET['settings-updated'] ) && get_option( 'cf7_to_spreadsheet_google_code' ) != '' ) {
-				include_once( plugin_dir_path( __FILE__ ). "lib/class-google-spreadsheet.php" );
+				require ( plugin_dir_path( __FILE__ ). "lib/class-google-spreadsheet.php" );
 				//call method to preauthontication
 				google_spreadsheet::google_pre_authentication( get_option( 'cf7_to_spreadsheet_google_code' ) );
 				update_option( 'cf7_to_spreadsheet_google_code', null );
 			}
 		}
 
 		/**
+		* Function Name: spreadsheet_editor_panel
+		* Function Description: Saving google spreadsheet data
+		* @param object $post
+		*/
+
+		public function spreadsheet_editor_panel( $post ) { 
+			ob_start();
+		    include_once ( plugin_dir_path( __FILE__ ).'include/cf7-editor-panel-html.php' );
+		    $editor_panel_html = ob_get_contents();
+			ob_end_clean();
+		    echo $editor_panel_html;
+		}
```

Comment:
@Abhishilawat Indentation here. make sure to format the code throughout the plugin.